project/config/victimspaymentsboard/config/core.extension.yml

```diff
@@ -66,7 +66,6 @@ module:
   layout_builder: 0
   layout_discovery: 0
   link: 0
-  link_fix_absolute_urls: 0
```

Comment:
Please leave this module installed, it should be installed on all Unity sites as it converts absolute URLs that reference other pages within the site to relative URLs.